python/ray/dashboard/modules/node/node_head.py

```diff
@@ -403,31 +404,39 @@ async def _update_node_stats(self):
             return_exceptions=True,
         )
 
-        for node_info, reply in zip(nodes, replies):
-            node_id, _ = node_info
-            if isinstance(reply, asyncio.CancelledError):
-                pass
-            elif isinstance(reply, grpc.RpcError):
-                if reply.code() == grpc.StatusCode.DEADLINE_EXCEEDED:
-                    logger.exception(
-                        f"Cannot reach the node, {node_id}, after timeout {TIMEOUT}. "
-                        "This node may have been overloaded, terminated, or "
-                        "the network is slow."
-                    )
-                elif reply.code() == grpc.StatusCode.UNAVAILABLE:
-                    logger.exception(
-                        f"Cannot reach the node, {node_id}. "
-                        "The node may have been terminated."
-                    )
-                else:
+        new_node_stats = {}
+
+        def postprocess():
```

Comment:
can we make `postprocess` a pure function: accepts (nodes, replies) and returns new_node_stats